staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -125,7 +125,7 @@ func (a *AdmissionOptions) ApplyTo(
 		pluginNames = a.enabledPluginNames()
 	}
 
-	pluginsConfigProvider, err := admission.ReadAdmissionConfiguration(pluginNames, a.ConfigFile, scheme)
+	pluginsConfigProvider, err := admission.ReadAdmissionConfiguration(pluginNames, a.ConfigFile, configScheme)
```

Comment:
thanks. Which brings us back to the observation that the testing of all of this is too weak.

Reply:
@hzxuzhonghu if you have capacity. improving the test coverage here would be awesome!

Reply:
I would like to.